pages/index.tsx

```diff
@@ -99,9 +118,41 @@ const HeaderSection = (props: HomePageHeader) => {
           </Typography>
         </Box>
       </Box>
-      <Typography variant="body_normal" color="#fff" sx={{ maxWidth: 950, mx: 'auto' }}>
-        {props.subtitle}
```

Comment:
can you help me understand the motivation for changing from this `props.subtitle` variable to having the text/copy directly inline? is it because you want to do further styling here that can't easily be done in the props data structure?

Reply:
It looks like it uses JSON-LD structured data in PageHead.tsx to provide this meta data for the page